### ISSUE TITLE
Some Characters such as ☆ and ♪ rendering as "▯"

### ISSUE THREAD

**Issue by wdpm** [sumatrapdfreader/sumatrapdf#2890]
## Reading environment:
-  sumatraPDF version: v3.5.15237 64bit pre-release
-  [last commit 8cc46a2](https://github.com/sumatrapdfreader/sumatrapdf/commit/8cc46a222e393a56f71918ec262ee8eaff2e08fd)
- OS: Windows 10

## Sample epub file:
[demo_novel.zip](https://github.com/sumatrapdfreader/sumatrapdf/files/9403191/demo_novel.zip)

## Screenshots
![image-20220823213246351](https://user-images.githubusercontent.com/14016242/186179309-66edaff3-3649-416b-9748-0dee11d6be92.png)

## Comparison of other readers
> Sigil 1.9.10
![image-20220823213440483](https://user-images.githubusercontent.com/14016242/186179711-cd958ece-a311-4ef1-bfd5-242cdee3cbb6.png)

> calibre 5.43 64bit
![image-20220823213531452](https://user-images.githubusercontent.com/14016242/186179729-6405d0b5-9868-4511-a77f-1c3a763ec0e2.png)

Hope to fix this issue.

**Comment by wdpm** [comment]
@eladkarako Thanks for your reply.
But I have two points to explain：
1. regarding this file [demo_novel.zip](https://github.com/sumatrapdfreader/sumatrapdf/files/9403191/demo_novel.zip), it does not have any fonts set in the source code of epub sample file.
2. I try your configuration metioned above
```
EbookUI [ 
	FontName = Noto Sans
 ]
```
in SumatraPDF-Setting.txt. (And I make sure I have installed Noto Sans font in my OS). After I save the configuration and restart the SumatraPDF App to check whether it works. But unluckily it does not work. 
Then I re-check the configuartion file. I noted this thing in the bottom of file:

```
# Settings below are not recognized by the current version
EbookUI [
	FontName = Noto Sans
]
```
Notes: I try the tests above in both the latest stable SumatraPDF 3.4.6-64 and the latest pre-release SumatraPDF v3.5.13253.

---
Then I visit the [setting manual link](https://www.sumatrapdfreader.org/settings/settings3-5) and want to find any descriptions about EbookUI. Finally the result is None. I want to know whether it's not ready now for `EbookUI` setting?

**Comment by GitHubRulesOK** [comment]
SumatraPDF ebook fonts only apply to some legacy ebook formats so newer versions don't apply them to ePub

You can try older versions or see if the epub may be bettered, currently there are no reliable ways to over-ride internal values

![image](https://user-images.githubusercontent.com/12754913/195637473-a37808f8-d063-45e2-94d3-64a20eefb3ba.png)

**Comment by wdpm** [comment]
@eladkarako 
I found I made a big mistake 
> regarding this file [demo_novel.zip](https://github.com/sumatrapdfreader/sumatrapdf/files/9403191/demo_novel.zip), it does not have any fonts set in the source code of epub sample file.

very sorry about this, I set a font named LXGW WenKai Screen but I doesn't include it in this epub sample.

I test your edited epub file [demo_novel_with_embedded_fonts.epub.zip](https://github.com/sumatrapdfreader/sumatrapdf/files/9778471/demo_novel_with_embedded_fonts.epub.zip), it works well.

![Snipaste_2022-10-14_00-10-58](https://user-images.githubusercontent.com/14016242/195649639-91c17254-3fef-4dbb-a570-3d144d559a7c.png)

The conclusion is I must include correct css link in html and embed custom font(prefer using local(...), then url(...)) to cover specifial symbols like ♪ or ☆ when using SumatraPDF to view epub format.

---

@GitHubRulesOK 

> SumatraPDF ebook fonts only apply to some legacy ebook formats so newer versions don't apply them to ePub
> You can try older versions or see if the epub may be bettered, currently there are no reliable ways to over-ride internal values

Now I know SumatraPDF currently has no reliable ways to over-ride internal values. I will try the solution above provided by @eladkarako if I want to satisfy SumatraPDF users.

Finally, thank you(@eladkarako @GitHubRulesOK ) for your time in troubleshooting this issue.😊
